tests/cloudpickle_test.py

```diff
@@ -887,6 +891,107 @@ def f1():
                                         clone_func=clone_func)
             assert_run_python_script(textwrap.dedent(code))
 
+    def test_closure_interacting_with_a_global_variable(self):
+        global _TEST_GLOBAL_VARIABLE
+        orig_value = _TEST_GLOBAL_VARIABLE
```

Comment:
maybe add:

```python
assert _TEST_GLOBAL_VARIABLE == "default_value"
```

to make the test both more easy to follow and robust to side effects by other tests.